### ISSUE TITLE
RGB vs Alpha Range

### ISSUE THREAD

**Issue by meltingice** [pkrumins/node-png#15]
I'm not sure if it makes sense that the alpha channel is inverted. What was the reasoning behind this?

If anything, this should be made more obvious in the documentation. It took me awhile to figure out what was going wrong.

**Comment by m0ose** [comment]
I agree. That is really weird. The alpha channel should be transparent at 0 and opaque at 255
